pulp_smash/tests/rpm/api_v2/test_schedule_consumer.py

```diff
@@ -0,0 +1,163 @@
+# coding=utf-8
+"""Test that verify scheduled RPM installation and un-installation."""
+import datetime
+import random
+import time
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, cli, config, exceptions, selectors, utils
+from pulp_smash.constants import (
+    CONSUMERS_PATH,
+    REPOSITORY_PATH,
+    RPM_UNSIGNED_FEED_URL,
+)
+from pulp_smash.tests.rpm.api_v2.utils import gen_distributor, gen_repo
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+REQUIRED_SERVICES = ('goferd',)
+
+REQUIRED_PACKAGES = (
+    'pulp-agent',
+    'pulp-consumer-client',
+    'pulp-rpm-consumer-extensions',
+    'pulp-rpm-handlers',
+    'pulp-rpm-yumplugins',
+    'python-gofer-qpid',
+)
+
+
+class ScheduledTaskConsumer(unittest.TestCase):
+    """Verify scheduled RPM installation and un-installation work.
+
+    Test the installation and un-installation of scheduled package in a
+    consumer.
+    This test case targets:
+    * `Pulp #2680 <https://pulp.plan.io/issues/2680>`_
```

Comment:
This isn't a list:

```
Some prose.
* List.
* Items.
```

This is:

```
Some prose.

* List.
* Items.
```

Run `make docs-html` if you're unsure what something will do, or whether the code in question acts correctly.